### ISSUE TITLE
Generator outputs are non-sense after 5 days of training

### ISSUE THREAD

**Issue by adrienchaton** [eps696/stylegan2ada#14]
Hi,

I was using the official StyleGAN2-ada from NVIDIA's pytorch repo, which worked well but do not allow to directly train on rectangular images. So I am trying this repo (thanks for the work !) on 768x1024 images.

The saved _reals.jpg images look totally fine. But after 4368kimg the generator is still outputting "crap". See the output of fake-4368.jpg below:

![fake-4368](https://user-images.githubusercontent.com/35500385/157416211-068518e7-b282-4265-9778-c4675b70c38d.jpg)

Has anyone faced this please ? The only configuration I provide to the training is --batch 8 --kimg 10000 and the dataset is about 40k images all cropped to 768x1024. I have inspected the dataset, its curated and of consistent quality.

Thanks for any hints, would be nice not to spend too much runs into black and green glitches .. (the targets are natural images)

**Comment by eps696** [comment]
that's weird. would you share few more samples from different stages of the training? 
did you train from scratch or finetuned some model?

**Comment by adrienchaton** [comment]
Thanks for your reply, I trained from scratch.
It seems that earlier outputs are plausible and around 3568kimgs it goes nuts ..
I will try to restart training from a checkpoint a little bit before that.
Would you have any thought on batch size or other hyper-parameter value I could tweak please ?

**Comment by eps696** [comment]
if it went ok'ish till some point and then got broken, i'd restart it from before that point. no reason to "train" more after the model collapsed.. 
batch size is better be kept as big as possible [depending on your gpu ram]. i'd decrease learning rate a bit. you may also play with `gamma` hyperparam, it's known to help stabilizing training process for too various datasets.

also you can try adapting some pretrained model to your resolution and finetuning it with your data. there are corresponding scripts/commands for that in both colab and local versions.

**Comment by eps696** [comment]
[closed for inactivity]
